src/components/charts/WaterStats.js

```diff
@@ -0,0 +1,128 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import SubmitButton from '../SubmitButton';
+
+export default function WaterStats({ filteredCardsByTime }) {
+  const [water, setWater] = useState('');
+
+  const filteredCardsByWater = filteredCardsByTime.filter(card =>
+    card.water.includes(water)
+  );
+
+  const allCatchesInManyArrays = filteredCardsByWater.map(object => {
+    const tempArray = object.catches.map(entry => {
+      return entry.species;
+    });
+    return tempArray;
+  });
+
+  const allCatches = allCatchesInManyArrays.flat();
```

Comment:
I'm not sure if this would work, but could give it a try :) 
```suggestion
  const allCatches = filteredCardsByWater.map(object => {
    const tempArray = object.catches.map(entry => {
      return entry.species;
    });
    return tempArray.flat();
  });
```

Reply:
nice thought. Does not work unfortunately